make/RunTests.gmk

```diff
@@ -868,6 +869,13 @@ define SetupRunJtregTestBody
 
   $1_JTREG_BASIC_OPTIONS += -e:TEST_IMAGE_DIR=$(TEST_IMAGE_DIR)
 
+  ifneq ($$(DOCS_IMAGE_DIR), )
+    DOCS_PATH := $$(DOCS_IMAGE_DIR)
+  else
+    DOCS_PATH := $$(OUTPUTDIR)/images/docs
+  endif
+  $1_JTREG_BASIC_OPTIONS += -e:DOCS_IMAGE_DIR=$$(DOCS_PATH)
```

Comment:
Looking at this again, I just realized we do have this path defined in spec.gmk already, but with a different name: `DOCS_JDK_IMAGE_DIR` (along with `DOCS_JAVASE_IMAGE_DIR` and `DOCS_REFERENCE_IMAGE_DIR`). I think we should stick with this existing name instead of introducing a new one, so it's consistent across the whole build system. Then we don't need this fallback to $(OUTPUTDIR) anymore. `DOCS_JDK_IMAGE_DIR` will either be defined in spec.gmk from configure or from jib-profiles when running `run-tests-prebuilt-docs`.

Reply:
Thanks, I hadn't seen this